### DISCUSSION TITLE
Vscode ?

### DISCUSSION THREAD

**Discussion by FirstToGlitch** [Yisus7u7/termux-desktop-xfce#81] [Q&A] [answered]
Can VSCode be added as an editor? Asking since it is available for Linux armhf in the official website.

https://code.visualstudio.com/docs/?dv=linuxarmhf_deb

also going to that link automatically downloads it.

**Comment by JyanJohn** [comment, marked as answer]
Termux cant run vscode.

If you want vscode on your phone I suggest installing it on proot-distro

+-----------------------+
INSTALLING A DISTRO

1. apt update

2. pkg install proot-distro -y

3. proot-distro install ubuntu

4. proot-distro login ubuntu
+-----------------------+
SETTING UP UBUNTU

1. apt update && apt upgrade

2. apt install sudo -y

3. adduser (yourname)
(NOTE: after doing this just complete the questions to add the user)

(Since the most stable vsocde for android is code-oss, the package will be from the Kali Repository)

4. apt install nodejs -y

5. wget http://http.kali.org/pool/main/c/code-oss/code-oss_1.56.1-0kali3_arm64.deb

6. dpkg -i code-oss_1.56.1-kali3_arm64.deb

7. exit
+----------------------+
IN TERMUX
Fire up Termux-Desktop-XFCE

After inside the GUI open a terminal and follow these command

xhost+

proot-distro login ubuntu --user (yourname) --shared-tmp

export DISPLAY=:1

code-oss
+----------------------+
